clash_royale_service.py

```diff
@@ -33,12 +34,16 @@ def compare_card_levels(p1, p2):
     return compare_result
 
 
-class ClanMembersRanker:
+class ClashRoyaleService:

```

Comment:
You also need to do this renaming in the discord commands or main .py files too don't you? Otherwise existing commands would fail